dashboard/config/routes.rb

```diff
@@ -79,12 +79,24 @@ module OPS
   get '/u/:id', to: redirect('/c/%{id}')
   get '/u/:id/:action_id', to: redirect('/c/%{id}/%{action_id}')
 
+  # These links should no longer be created (August 2017), though we will continue to support
+  # existing links. Instead, create /r/ links.
```

Comment:
You are correct. By "no longer created", I mean no longer sent to the user. Let me improve the wording in another commit.

You aren't missing something, /c/ links will continue to work. For better or for worse, Product doesn't believe in sunsetting links (note that /s/ and /sh/ redirect to /c/).

As for the loophole, Product seems to consider this the best solution possible.